vtex-apps/google-geolocation-resolver
# CHK-435: Session token

#### What problem is this solving?

Resolves query to retrieve the session token and handle when session tokens are used on search arguments.

Depends on https://github.com/vtex-apps/geolocation-graphql-interface/pull/6.

#### How should this be manually tested?

[Run this GraphQL query.](https://w0geo--checkoutio.myvtex.com/_v/private/vtex.geolocation-graphql-interface@0.1.0/graphiql/v1?query=%7B%0A%20%20sessionToken%0A%7D%0A)

The session token handling part is quite hard to test, as there is no easy way to tell that the session tokens are being used correctly. Check the Jira ticket comment for ways to test it in the long run.

#### Checklist/Reminders


#### Screenshots or example usage

#### Notes

[Google suggests using UUID](https://developers.google.com/places/web-service/session-tokens#create-session-tokens), but any unique token would be okay as well.

Also, I suggest reviewing per commit.
